index.html

```diff
@@ -0,0 +1,299 @@
+                    </li>
+                    <li class="service-item servise__list-item">
+                        <div class="circle-icon service-item__icon">
+                            <img class="service-item__icon-img" src="./resources/images/home.svg">
+                        </div>
+                        <div class="service-item__article">
+                            <h3 class="h3-like service-item__title">Cras ornare</h3>
+                            <p lang="en" class="prh service-item__prh">Simasellus ultrices nulla quis nibh. Quisque a lectus. Donec consectetuer ligula vulputate sem tristique</p>
+                            <div class="service-item__more"><a class="link" href="#">Read More<span class="service-item__more-icon icon-arrow-circle-right"/></a></div>
+                        </div>
+                    </li>
+                </ul>
+            </section>
+            <section class="about-us">
+                <ul class="about-us__staff-list"> 
+                    <li class="person about-us__person">
+                        <img class="person__photo" src="./resources/images/person1.png">
+                        <div class="person__char">
+                            <h3 class="h3-like person__title">Lorem ipsum dolor sit amet</h3>
+                            <p lang="en" class="prh person__prh">
+                                Sed egestas, ante et vulputate volutpat, eros pede semper est, vitae luctus metus
+                                libero eu augue. Morbi purus libero, faucibus adipiscing, commodo quis, gravida id,
+                                est. Sed lectus tpat, lacus a ultrices sagittis, mi neque euismod dui, eu pulvinar nunc sapien ornare nisl. Phasellus pede arcu, dapibus eu
+                            </p>
+                            <button class="btn btn--narrow person__btn-more">MORE</button>
+                        </div>
+                    </li>
+                    <li class="person about-us__person">
+                        <img class="person__photo" src="./resources/images/person2.png">
+                        <div class="person__char">
+                            <h3 class="h3-like person__title title--last">Sed egestas ante et vulpu</h3>
+                            <p lang="en" class="prh person__prh">
+                                Sed egestas, ante et vulputate volutpat, eros pede semper est, vitae luctus metus
+                                libero eu augue. Morbi purus libero, faucibus adipiscing, commodo quis, gravida id,
+                                est. Sed lectus tpat, lacus a ultrices sagittis, mi neque euismod dui, eu pulvinar nunc sapien ornare nisl. Phasellus pede arcu, dapibus eu
+                            </p>
+                            <button class="btn btn--narrow person__btn-more">MORE</button>
+                        </div>
+                    </li>
+                </ul>
+                <div class="tab about-us__tab">
```

Comment:
I consider this block as something separate from "about us" section. It is just some sidebar information. There is one HTML elements that matches better for that purpose. :)
Also I think that it's worth to wrap 2 elements up with one parent. It will be better from semantic point of view.